frontend/gist-frontend/src/Components/CreateUserComponent.tsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import { Form, FormField, Button, Box, Layer, Heading, Select } from "grommet";
+import { User, UserFemale, UserManager } from "grommet-icons";
+
+interface ICreateUserComponentProps {}
+
+interface ICreateUserComponentState {
+  showPictureModal: boolean;
+  image: string;
```

Comment:
instead of `state{image:string}` could we do `state{image : JSX.Element}` and just store the element here? i think it would get rid of a lot of the `if(image === "UserFemale")` etc. We don't have images set up yet on the backend, so we don't have to worry about storing an image string yet. We could just have the icons as the only choices for now.

Reply:
Yeah, this makes a lot of sense. I previously was doing that, but for some reason reverted to strings and have no idea why.